src/System.IO.Pipelines/src/System/IO/Pipelines/ThreadPoolScheduler.NetCoreApp21.cs

```diff
@@ -12,31 +12,15 @@ internal sealed class ThreadPoolScheduler : PipeScheduler
         public override void Schedule(Action action)
         {
             // Queue to low contention local ThreadPool queue; rather than global queue as per Task
-            System.Threading.ThreadPool.QueueUserWorkItem(_actionWaitCallback, action, preferLocal: true);
+            System.Threading.ThreadPool.QueueUserWorkItem(s_actionCallback, action, preferLocal: true);
         }
 
         public override void Schedule(Action<object> action, object state)
         {
             // Queue to low contention local ThreadPool queue; rather than global queue as per Task
-            System.Threading.ThreadPool.QueueUserWorkItem(_actionObjectWaitCallback, new ActionObjectAsWaitCallback(action, state), preferLocal: true);
+            System.Threading.ThreadPool.QueueUserWorkItem(action, state, preferLocal: true);
```

Comment:
@davidfowl, @pakrym, just FYI that I changed this this morning in order to get the coreclr update through into corefx.  Hopefully this is ok.